### PR TITLE
fix invalid link flutter.dev

### PR DESCRIPTION
**Pull request by nickf2k**
fix invalid flutter.dev link because we no longer use it

fix issue https://github.com/flutter/website/issues/11294

## Presubmit checklist

- [ ] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by sfshaza2 — COMMENTED** [review]
Hi, @nickf2k! Can you tell me why you find this change to be needed? As it's written now, the link takes me to the correct page.

**Comment by parlough** [comment]
> Can you tell me why you find this change to be needed? As it's written now, the link takes me to the correct page.

It's only working due to a temporary redirect we added to avoid breaking links from the `flutter` tool.

Now the `flutter` tool will output `https://flutter.dev/to/macos-android-setup` in this case.

File: src/_includes/docs/install/compiler/android.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ location of your Java installation.
 Android sdkmanager tool was found, but failed to run
 (/Users/atsansone/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager): "exited code 1".
 Try re-installing or updating your Android SDK,
-visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions.
+visit https://docs.flutter.dev/get-started/install/macos/mobile-android#configure-android-development for detailed instructions.
```
Comment (parlough) [inline src/_includes/docs/install/compiler/android.md:179]: Can you update this to match the new link from the `flutter` tool? As this is an example of the output. Thanks!

```suggestion
visit https://flutter.dev/to/macos-android-setup for detailed instructions.
```

Comment (nickf2k) [reply src/_includes/docs/install/compiler/android.md]: Sure. Thank you

**Review by parlough — APPROVED** [review]
Thank you for catching and fixing this! It's appreciated :D

**Pull request merged**